### PR TITLE
Feature/delete game

### PR DESCRIPTION
**Pull request by tabbykatz**
## Overview

- create db and express functionality to handle delete
- useApi can send a DELETE method via fetch
- Games component 'Remove' button removes a game from user's collection, reloads Games, and notifies with toast
## Feedback Requested

_ This is a simple addition but any feedback welcome._

### CODE REVIEW

**Comment by ZelmaSedano** [comment]
Looks great!

File: app/src/auth/useApi.js [hunk 1]
```diff
@@ -7,9 +7,14 @@ const makeApi = (accessToken) => {
     getGames: () => _get("/api/users/games"),
     findGames: (name) => _get("/api/games", { name }),
     addGame: (game) => _post("/api/users/games", { game }),
+    deleteGame: (id) => _delete(`/api/users/games/${id}`, { id }),
     addOrUpdateUser: (user) => _post("/api/users", { user }),
   };
 
+  const _delete = (url, id) => {
+    _fetch(url, { method: "DELETE" });
```
Comment (talarsen) [inline app/src/auth/useApi.js:15]: clean code on this function!

**Review by talarsen — COMMENTED** [review]
Looks like clean code to me. I hope it is going well.

**Comment by tabbykatz** [comment]
thanks, everyone! 😊

**Pull request merged**